### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by tdivis**

### CODE REVIEW

File: docker/Dockerfile.specific_proxy_service [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       python3-pip python3-setuptools python3-dev \
```
Comment (tpazderka) [inline docker/Dockerfile.specific_proxy_service]: It is a convention to put one dependency per line.

File: docker/specific_proxy_service_cfg.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+PROXY_SERVICE_RESPONSE_TOKEN = {
+    'HASH_ALGORITHM': 'sha256',
+    'SECRET': 'mySecretProxyserviceResponse',
+    'ISSUER': 'specificCommunicationDefinitionProxyserviceResponse',
+}  # type: Dict[str, str]
+
+PROXY_SERVICE_IDENTITY_PROVIDER = {
+    'ENDPOINT': 'https://tnia.eidentita.cz/fpsts/saml2/basic',
+    'REQUEST_ISSUER': 'http://pokuston.office.nic.cz:58909/saml/idp.xml',
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: Well, it's suppose to be example settings, but OK, i'll put this there.

File: docker/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# RUn in Docker
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/README.md]: Done

File: docker/specific_proxy_service_cfg.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        'host': 'pokuston-m-01.office.nic.cz',
+        'port': 10800,
+        'request_cache_name': 'nodeSpecificProxyserviceRequestCache',
+        'response_cache_name': 'specificNodeProxyserviceResponseCache',
+    }
+}  # type: Dict[str, Any]
+
+PROXY_SERVICE_EIDAS_NODE = {
+    'PROXY_SERVICE_RESPONSE_URL': 'http://pokuston.office.nic.cz:8888/EidasNode/SpecificProxyServiceResponse',
+    'REQUEST_ISSUER': 'http://pokuston:8888/EidasNode/ConnectorMetadata',
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: OK, removed from here.

File: docker/specific_proxy_service_cfg.py [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    'RESPONSE_ISSUER': '',
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: OK, removed.

[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: OK, set tot 'specific-proxy-service'

File: docker/specific_proxy_service_cfg.py [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+# Application definition
+INSTALLED_APPS = [
+    'django.contrib.staticfiles',
+    'eidas_node.proxy_service.apps.ProxyServiceConfig',
+]
+MIDDLEWARE = [
+    'django.middleware.common.CommonMiddleware',
+]
+ROOT_URLCONF = 'eidas_node.proxy_service.urls'
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: Added in comment.

File: docker/Dockerfile.specific_proxy_service (hunk 1 above)
Comment (tdivis) [reply docker/Dockerfile.specific_proxy_service]: Splitted and sorted.

File: docker/specific_proxy_service_cfg.py [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+        'django.request': {
+           'handlers': ['console'],
+           'level': 'DEBUG',
+           'propagate': False,
+        },
+        'django': {
+           'handlers': ['console'],
+           'level': 'DEBUG',
+           'propagate': False,
+        },
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: OK, added.

File: docker/specific_proxy_service_cfg.py [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+PROXY_SERVICE_RESPONSE_TOKEN = {
+    'HASH_ALGORITHM': 'sha256',
+    'SECRET': 'mySecretProxyserviceResponse',
+    'ISSUER': 'specificCommunicationDefinitionProxyserviceResponse',
+}  # type: Dict[str, str]
+
+PROXY_SERVICE_IDENTITY_PROVIDER = {
+    'ENDPOINT': 'https://tnia.eidentita.cz/fpsts/saml2/basic',
+    'REQUEST_ISSUER': 'http://pokuston.office.nic.cz:58909/saml/idp.xml',
+    'RESPONSE_ISSUER': 'urn:microsoft:cgg2010:fpsts',
```
[the remark this answers is not in the document]
Comment (tdivis) [reply docker/specific_proxy_service_cfg.py]: OK, removed.

File: docker/Dockerfile.specific_proxy_service (hunk 1 above)
Comment (tpazderka) [reply docker/Dockerfile.specific_proxy_service]: They are not sorted :)

Comment (tdivis) [reply docker/Dockerfile.specific_proxy_service]: Almost, now it should be OK :).

**Pull request merged**